### DISCUSSION TITLE
避免依赖冲突问题

### DISCUSSION THREAD

**Discussion by abstinencetang** [arextest/arex-agent-java#508] [Q&A]
为了避免第三方依赖冲突，将arex-instrumentation-foundation模块下的com.github.luben.zstd依赖通过maven-shade-plugin方式改包名后出现UnsatisfiedLinkError异常，有遇到过这个问题么？其他依赖改包名是没问题的

**Comment by mr3** [comment]
@abstinencetang Thank you for posting in the AREX community! We value your feedback and our team is paying attention to the feedback, feature requests & bugs we are hearing from users in the Community Discussions.

We'll check this out as soon as we can
